community/pmount/build

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh -e
+
+patch -p1 < 001-disable-intltool.patch
+patch -p1 < 001-fix-musl-build.patch
+
+# autoconf warnings are treated as errors.
+set +e
+automake
+autoconf
+set -e
```

Comment:
We should never really set +e. If autotools wants to be broken, we should let it :X 
`autoreconf -sfi` is what we should do instead, though presumably this is responsible for me having to modify the `Makefile` after it is generated.